README.md

```diff
@@ -34,24 +34,38 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is just HTML with meaning.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+display: inline;   pushes everything on the same line while,
+display: block;   starts on a new line.
```

Comment:
✔️ 
Don't forget about the inline items not responding to top/bottom properties.